src/terminal/parser/stateMachine.hpp

```diff
@@ -129,7 +133,9 @@ namespace Microsoft::Console::VirtualTerminal
             DcsIntermediate,
             DcsParam,
             DcsPassThrough,
-            DcsTermination
+            DcsTermination,
+            SosPmApcString,

```

Comment:
OK this is like the very definition of developer-laziness. But in my defense, this is also what https://vt100.net/emu/dec_ansi_parser is using.

Reply:
I think when SOS/PM/APC actually do something on their own, it would still be easy to create new dedicated states for them.